packages/accounts-password/password_tests.js

```diff
@@ -1697,6 +1697,34 @@ if (Meteor.isServer) (function () {
     }
   )
 
+  Tinytest.add(
+    'passwords - enroll tokens don\'t get cleaned up when reset tokens are cleaned up',
+    function (test) {
+      var email = test.id + '-intercept@example.com';
+      var userId = Accounts.createUser({email: email, password: 'password'});
+      Accounts.sendEnrollmentEmail(userId, email);
+      test.isTrue(!!Meteor.users.findOne(userId).services.password.reset);
```

Comment:
I did it mostly to stay consistent with the rest of the tests on token cleanup. I have looked into it and I don't see why use `!!`. `Meteor.users.findOne(userId).services.password.reset` returns an object or `undefined`. At first I thought it was to handle empty object case `{}`. Though both cases `test.isTrue({})` and `test.isTrue(!!{})` are true. Both cases `test.isTrue(undefined)` and `test.isTrue(!!undefined)` are false. I can remove the `!!`. What do you think?